app/types.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-typography';
```

Comment:
What is the idea behind this module declaration? I was having some trouble using the 'react-typography' module, was this needed to make it work?